libr/bin/p/elf.mk

```diff
@@ -3,7 +3,7 @@ OBJ_ELF+=../format/elf/elf.o ../format/elf/elf_write.o
 
 STATIC_OBJ+=${OBJ_ELF}
 TARGET_ELF=bin_elf.${EXT_SO}
-LINK+=-L../../db -lr_db
+LINK+=-L../../db -lr_db -L../../io -lr_io -L../../cons -lr_cons
```

Comment:
scary! why do you need r_cons and r_io here? you should be able to use bin.bio instead